### PR TITLE
WaveunitNotFoundError_msg_changed

### LINKED ISSUE #1342: database complains of EIT non-existent wavelengths units
Without any reason I've been able to find, `database.fetch` fails with some EIT files... not always the same, with the following error message:
```
WaveunitNotFoundError: the wavelength unit cannot be found in ~/sunpy/data/efz20000912.203610
```
That was from this query:
```python
db.fetch(vso.attrs.Time('2000/9/12T20:30:00', '2000/9/12T22:00:00'),
vso.attrs.Instrument('EIT'))
```

### PR DESCRIPTION
**Pull request by dumindux**
closes #1342

### CODE REVIEW

File: sunpy/database/tables.py [hunk 1]
```diff
@@ -48,7 +48,8 @@ def __init__(self, obj):
         self.obj = obj
 
     def __str__(self):  # pragma: no cover
-        return 'the wavelength unit cannot be found in {0}'.format(self.obj)
+        return 'the wavelength unit cannot be found in {0}'.format(self.obj) + \
+               ' and deffault_waveunit not specified when opening the database'
```
Comment (Cadair) [inline sunpy/database/tables.py]: deffault should be default.

Comment (dumindux) [inline sunpy/database/tables.py]: oops I am sorry. Copied and pasted from the comment :p.

**Comment by dumindux** [comment]
Last time Travis build was successful. This time it fails. Not sure why.

**Comment by dumindux** [comment]
Not sure why Travis build is failing. It was successful in the early commit.

**Comment by wafels** [comment]
This PR is fine - the glob fix of #1348 should fix the failures. :+1: pending the the acceptance of #1348

**Comment by ehsteve** [comment]
:+1:

**Comment by Cadair** [comment]
:+1:

**Pull request merged**